docker-standalone-ethereum-studio-on-ubuntu/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Standalone Ethereum Studio",
+  "description": "This template deploys a docker with standalone version of Ethereum Studio on Ubuntu.",
+  "summary": "This template deploys a docker with standalone version of Ethereum Studio on Ubuntu.",
+  "githubUsername": "ether-camp",
```

Comment:
Who will be maintaining this template going forward? Is this someone from ether-camp or you?